app/bundles/CoreBundle/Resources/views/Helper/publishstatus_badge.html.twig

```diff
@@ -7,5 +7,6 @@
         warning
     {%- endif -%}
 {% endset %}
-{% set labelText = ('mautic.core.form.' ~ entity.getPublishStatus())|trans %}
-<h4 class="fw-sb"><span class="tt-u label label-{{ labelColor }}">{{ labelText }}</span></h4>
\ No newline at end of file
+{% set status = entity.getPublishStatus() %}
+{% set labelText = ('mautic.core.form.' ~ (status == 'published' ? 'active' : 'inactive'))|trans %}
+<span title="{{ labelText }}" aria-label="{{ labelText }}" class="pa-10 bg-{{ labelColor }} mt-3 label label-{{ labelColor }}"> </span>
```

Comment:
I'd prefer to show the text so people would know what this red square means, but a title is good enough I guess.

Reply:
The problem is that some entities are generic
So it’ll display “Active”, for example, for something you create to be available

this inconsistency could cause confusion, but the title is there to indicate Active or Inactive if someone gets in doubt about what it means 

Not a perfect solution, but I’m technically unable to solve the entities differentiation

Reply:
The label is already in the title and that's important for accessibility. So if it's good for visually impaired users then it should be OK for every one else, right?

Another idea: How about to use some icons here? Something like play/pause icons.

Reply:
Here we go

![image](https://github.com/mautic/mautic/assets/116097999/74e9eb8c-c670-4aa9-af21-0b38f55d9fe6)

![image](https://github.com/mautic/mautic/assets/116097999/b81e401d-8625-4223-928c-180be7af6dd5)